rest-api/src/main/resources/definitions/enums/notification_protocol.yml

```diff
@@ -0,0 +1,12 @@
+type: string
+description: |
+    Supported notification protocols.
+
+    |Protocol|Description|
+    |---|---|
+    |application|Push notifications, using an endpoint ARN as generated by SNS.|
+    |sms|SMS notifications, using a (as of now) US phone number.|
```

Comment:
Is it actually limited? I hope it is...

Reply:
You're right. I'm not sure where I got the idea that it was US phone numbers only. Apparently, as long as the country code is included, we can send anywhere. (And I think Phone already encapsulates that.) I'll fix the documentation.